sources/packages/web/src/views/student/financial-aid-application/FullTimeApplication.vue

```diff
@@ -254,6 +255,14 @@ export default {
       }
     };
 
+    const customEventCallback = async (form: any, event: any) => {
+      if ("routeToStudentProfile" === event.type) {
```

Comment:
maybe we can add enums for custom callback names in vue side

Reply:
Enum created.